Markdown/MultiMarkdown.sublime-syntax

```diff
@@ -1,29 +1,37 @@
 %YAML 1.2
 ---
-# http://www.sublimetext.com/docs/3/syntax.html
-version: 2
 name: MultiMarkdown
-first_line_match: (?i)^format:\s*complete\s*$
 scope: text.html.markdown.multimarkdown
+
+extends: Packages/Markdown/Markdown.sublime-syntax
+
+first_line_match: (?i:^format:\s*complete\s*$)
+
 variables:
-  header: ((?=[A-Za-z0-9])[\w -]+)(:)
+  header: ([A-Za-z0-9][\w -]*)(:)
+
 contexts:
   main:
-    - match: '^{{header}}\s*'
+    - match: ^(?={{header}})
+      push: multimarkdown-header
+    - match: ^
+      set: multimarkdown-content
+
+  multimarkdown-header:
+    - match: ^$
+      pop: 1
```

Comment:
Since this is v1, it should probably be `pop: true`.

Reply:
We decided to move ont to `pop: 1` not after switching to `version 2´ syntaxes and to use the same style in the whole file. I don't see any reason why we should use `pop: true` here.